contracts/contract_life_cycle.md

```diff
@@ -0,0 +1,3 @@
+[back](./contracts.md)
+## Contract life cycle examples
```

Comment:
Shall this file be suppressed? I see "contracts/contracts.md" links to this doc though the lifecycle example is there.

Reply:
The plan is to add more examples.